build_tools/azure/install.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 set -e
+set -x
```

Comment:
was this just for debugging or do we still need it?

Reply:
It was for debugging, but we have it in a few other scripts and I find it easier to debug whenever an issue happens, instead of putting it there every time we need it.